win-odbc64/sql/cli/sqlcli.h

```diff
@@ -425,6 +433,8 @@ enum SQLTYPE_CODE {
     SQLTYPECODE_BITVAR   = 15   /* not supported */
 };
 
+#define SQL_BOOLEAN			13
+
 /* #ifndef SQLDTCODE_DATE */
 /* specifies the type of datetime data type */
 enum SQLDATETIME_CODE {

```

Comment:
@selvaganesang Thanks for reminder. We shouldn't use 13 to define a new datatype here and we also don't need define a new datatype . We need use SQLTYPECODE_BOOLEAN as the extend datatype.

@Weixin-Xu Please change it.